x/wasm-storage/keeper/filter.go

```diff
@@ -39,3 +60,55 @@ func ApplyFilter(filter []byte, reveals []RevealBody) ([]int, bool, error) {
 		return outliers, false, nil
 	}
 }
+
+func filterMode(dataPath string, exitCodes []uint8, reveals []string) ([]int, bool) {
```

Comment:
I think we should add a comment about what the function does and what it returns.

Reply:
here [8c49c59](https://github.com/sedaprotocol/seda-chain/pull/290/commits/8c49c59ff72231ea958fb1dd5e456622d9f10163)